rlenvs/Env.lua

```diff
@@ -34,7 +38,15 @@ end
 
 function Env:start()
     self.currentStep = 1
-    return self:_start()
+    local obs = self:_start()
+    return obs
+end
+
+function Env:render()
+    if self.qt and self.getDisplay then
+        self.window = self.window == nil and image.display({ image = self:getDisplay(), zoom = 10 }) or self.window
```

Comment:
`zoom` should be part of `opts` and handled by `Env` - 10 was hard-coded for Catch, but the default of 1 is appropriate for Atari.